tutorials/vscCodeNavigation.md

```diff
@@ -0,0 +1,99 @@
+
+To navigate to a symbol's definition, place your cursor on the symbol and press `F12`. Alternatively, right-click on the symbol and select **Go to Definition** from the context menu. If there are multiple definitions, you will be prompted to select the definition that you wish to navigate to.
+
+## Go to Super Implementation
+
+You can trace class implementations and overriding methods by hovering over the class or method and click **Go to Super Implementation** to navigate to its parent class or overridden method.
+
+![Go to Super Implementation](https://code.visualstudio.com/assets/docs/java/java-editing/goto-super.png)
+
+<video src="https://code.visualstudio.com/assets/docs/java/java-editing/goto-super-implementation.mp4" controls width="700px">Your browser does not support the video tag.</video>
+
+## Call Hierarchy
+
+You can view all calls from or to a function to understand the flow of your code and dependencies between functions.
+
+To view the call hierarchy of a function, right-click on the function and select **Peek > Peek Call Hierarchy** to open in an inline panel, or select **Show Call Hierarchy** to open in the sidebar.
+
+![Peek Call Hierarchy](https://code.visualstudio.com/assets/docs/java/java-editing/call-hierarchy.png)
+
+![Show Call Hierarchy](https://code.visualstudio.com/assets/docs/java/java-editing/call-hierarchy.gif)
+
+## Type Hierarchy
+
+You can explore the inheritance relationships between classes and interfaces in your code to understand and navigate complex type hierarchies between Java Objects.
+
+To view the call hierarchy of a class or interface, right-click on the class or interface and select **Show Type Hierarchy** to open in the sidebar.
+
+<video src="https://code.visualstudio.com/assets/docs/java/java-editing/type-hierarchy.mp4" controls width="700px">Your browser does not support the video tag.</video>
+
+## Folding Regions
+
+You can collapse or expand code blocks to focus on relevant sections of your code by clicking the folding icons in the gutter next to the line numbers or by using keyboard shortcuts:
+
+* **Fold** (`Ctrl+Shift+[`) folds the innermost uncollapsed region at the cursor.
+* **Unfold** (`Ctrl+Shift+]`) unfolds the collapsed region at the cursor.
+* **Toggle Fold** (`Ctrl+K Ctrl+L`) folds or unfolds the region at the cursor.
+* **Fold Recursively** (`Ctrl+K Ctrl+[`) folds the innermost uncollapsed region at the cursor and all regions inside that region.
+* **Unfold Recursively** (`Ctrl+K Ctrl+]`) unfolds the region at the cursor and all regions inside that region.
+* **Fold All** (`Ctrl+K Ctrl+0`) folds all regions in the editor.
+* **Unfold All** (`Ctrl+K Ctrl+J`) unfolds all regions in the editor.
+
+<video src="https://code.visualstudio.com/assets/docs/java/java-editing/folding-range.mp4" controls width="700px">Your browser does not support the video tag.</video>
```

Comment:
Add yourself as a contributor, at the bottom of the page (see other similar tutorials for guidance)